InstructLab_SLACK_GUIDE.md

```diff
@@ -37,17 +37,15 @@ We are an open, welcoming, and inclusive community and expect our members to be
 
 We take reports of harassment very seriously and will action any reports of inappropriate behavior as quickly as possible.
 
-To learn how to report abuse - and to whom you will be reporting - please see our [InstructLab slack Moderation Guide](https://github.com/instructlab/community/blob/main/InstructLabSlackModerationGuide.md).
+To learn how to report abuse - and to whom you will be reporting - please see our [InstructLab slack Moderation Guide](https://github.com/instructlab/community/blob/main/InstructLab_SLACK_MODERATION_GUIDE.md#workspace-administrators).
```

Comment:
sorry to nitpick, I don't think the `#workspace-administrators` anchor is needed here :sweat_smile: